### PR TITLE
support 16bit i8080 display bus

### PR DESCRIPTION
**Pull request by TychoVrahe**
Adds a support for 16bit display bus.

### CODE REVIEW

**Comment by TychoVrahe** [comment]
needs to wait for #2912

File: core/embed/rust/src/trezorhal/display.rs [hunk 1]
```diff
@@ -142,7 +150,7 @@ pub fn pixeldata(c: u16) {
 }
 
 #[inline(always)]
-#[cfg(not(all(feature = "model_tt", target_arch = "arm")))]
+#[cfg(not(any(feature = "disp_i8080_16it_dw", feature = "disp_i8080_8bit_dw")))]
```
Comment (mmilata) [inline core/embed/rust/src/trezorhal/display.rs:153]:
```suggestion
#[cfg(not(any(feature = "disp_i8080_16bit_dw", feature = "disp_i8080_8bit_dw")))]
```

**Comment by gat68** [comment]
Enviado desde mi Samsung Mobile de Telcel
-------- Mensaje original --------De: Martin Milata ***@***.***> Fecha: 26/4/2023  4:22 a. m.  (GMT-06:00) A: trezor/trezor-firmware ***@***.***> CC: Subscribed ***@***.***> Asunto: Re: [trezor/trezor-firmware] support 16bit i8080 display bus (PR #2928) 
@mmilata commented on this pull request.



In core/embed/rust/src/trezorhal/display.rs:
> @@ -142,7 +150,7 @@ pub fn pixeldata(c: u16) {
 }
 
 #[inline(always)]
-#[cfg(not(all(feature = "model_tt", target_arch = "arm")))]
+#[cfg(not(any(feature = "disp_i8080_16it_dw", feature = "disp_i8080_8bit_dw")))]

⬇️ Suggested change
-#[cfg(not(any(feature = "disp_i8080_16it_dw", feature = "disp_i8080_8bit_dw")))]
+#[cfg(not(any(feature = "disp_i8080_16bit_dw", feature = "disp_i8080_8bit_dw")))]

File: core/embed/rust/src/trezorhal/display.rs (hunk 1 above)
Comment (TychoVrahe) [reply core/embed/rust/src/trezorhal/display.rs]: https://github.com/trezor/trezor-firmware/pull/2928/commits/2e80bcdf62f6837e63f6332deee0dbad75c718f1

**Review by hiviah — APPROVED** [review]
LGTM. (The build segfault is weird though)

**Pull request merged**